raw_data/preprocessed/africa/guinea-bissau_portuguese.txt

```diff
@@ -0,0 +1,199 @@
+
+Participação de Infracções
+(47) Todos têm dever de informar as autoridades competentes quando verificarem a prática de infracções as disposições desta lei ou qualquer outra legislação ambiental, ainda que presuma que tais estejam na eminência de ocorrer.
+
+Seguro de responsabilidade Civil
+(48) Aqueles que exerçam actividades que envolvam alto grau de risco para o ambiente e como tal venham a ser classificados serão obrigados a assegurar a sua responsabilidade civil.
+
+CAPÍTULO VI - DAS PENALIZAÇÕES
+
+Tribunal Competente
+(49) [1] O conhecimento das acções a que se referem os artigos 43 e 44 da presente lei é da competência dos tribunais, conforme à lei de processo aplicável. [2] Nos termos do artigo 43 da presente lei os lesados têm a legitimidade para demandar os infractores nos tribunais para obtenção das correspondentes indemnizações. [3] Sem prejuízo da legitimidade dos lesados para propor as acções, compete ao Ministério Público a defesa dos valores protegidos por esta lei através, nomeadamente, dos mecanismos previstos na presente lei e demais aplicável.
+
+Crimes Contra o Ambiente
+(50) Além dos crimes previstos e punidos no Código Penal, serão ainda considerados crimes, as infracções que a legislação complementar vier a qualificar como tal de acordo com o disposto na presente lei.
+
+Crimes e Contravenções Ambientais
+(51) As infracções de carácter criminal, bem como as contravenções relativas ao ambiente, são objecto de regulamentação em legislação especial.
+
+Obrigatoriedade de Remoção das Causas da Infracção e da Reconstituição da Situação Anterior
+(52) [1] Os infractores são obrigados a remover as causas da infracção e a repor a situação anterior à mesma ou equivalente, salvo o disposto no número [3] deste artigo. [2] Se os infractores não cumprirem as obrigações acima referidas no prazo que lhes for indicado, as entidades competentes mandarão proceder às demolições das obras e acções necessárias à reposição da situação anterior à infracção, a expensas dos infractores. [3] Em caso de não ser possível a reposição da situação anterior à infracção, dos infractores ficam obrigados ao pagamento de uma indemnização nos termos gerais do direito e a realização das obras necessárias a minimização das consequências provocadas.
+
+CAPÍTULO VII - DAS DISPOSIÇÕES FINAIS E TRANSITÓRIAS
+
+Desenvolvimento da Lei de Bases do Ambiente
+(53) O Governo deve mediante legislação especial, regulamentar a presente lei, no prazo de um ano após a data da sua publicação, nomeadamente nos seguintes domínios: [a] Instituto Nacional do Ambiente; [b] Fundo Ambiental; [c] Inspecção Ambiental; [d] Laboratório de Referência Ambiental; [e] Organismos Vivos e Geneticamente Modificados/Biossegurança; [f] Acesso Equitativo dos Recursos e Partilha dos Benefícios; [g] Detecção Remota e Sistema de Informação Geográfica (DT&SIG); [h] Propriedade intelectual na área do ambiente (Fauna e Flora
```

Comment:
"Flora" at the end of the article should be followed by ").".

Reply:
I will update. Thanks.